courses/dl2/imdb_scripts/README.md

```diff
@@ -55,65 +55,96 @@ tok2id.py --prefix PREFIX [--max-vocab MAX_VOCAB] [--min-freq MIN_FREQ]
 
 ### 3. Fine-tune the LM
 
-First run `train_tri_wt.py` to create a pre-trained language model using WikiText-103 (or whatever base corpus you prefer)
-or use one of the pre-trained models.
+Before fine-tuning the language model, you can run `pretrain_lm.py` to create a 
+pre-trained language model using WikiText-103 (or whatever base corpus you prefer).
 
 Example command: `python train_tri_wt.py data/wiki/de/ 0 --lr 1e-3 --cl 12`
```

Comment:
I think this line should have `python pretrain_lm.py` instead of`python train_tri_wt.py`

Reply:
Thanks for pointing that out! Missed that! :)